griptape/drivers/vector/redis_vector_store_driver.py

```diff
@@ -15,6 +19,7 @@ class RedisVectorStoreDriver(BaseVectorStoreDriver):
     db: int = field(kw_only=True, default=0)
     password: Optional[str] = field(default=None, kw_only=True)
     index: str = field(kw_only=True)
+    vector_dimensions: int = field(kw_only=True, default=1536)
```

Comment:
Let's move this into `create_index` parameters, since it's only used there. Let's also not have a default, since embedding dimensions vary from model to model.